das/das.go

```diff
@@ -95,6 +99,8 @@ func OptionalAddressFromString(s string) (*common.Address, error) {
 func DataAvailabilityConfigAddOptions(prefix string, f *flag.FlagSet) {
 	f.Bool(prefix+".enable", DefaultDataAvailabilityConfig.Enable, "enable Anytrust Data Availability mode")
 
+	f.Duration(prefix+".request-timeout", DefaultDataAvailabilityConfig.RequestTimeout, "Data Availability Service request timeout duration")
```

Comment:
We need to set a sensible default timeout value here. (`DefaultDataAvailabilityConfig.RequestTimeout` is empty)

Reply:
Done.